src/coreclr/jit/fgbasic.cpp

```diff
@@ -1236,6 +1249,300 @@ void Compiler::fgFindJumpTargets(const BYTE* codeAddr, IL_OFFSET codeSize, Fixed
                                 break;
 #endif
 
+                            case NI_SRCS_UNSAFE_Add:
+                            case NI_SRCS_UNSAFE_AddByteOffset:
+                            case NI_SRCS_UNSAFE_AreSame:
+                            case NI_SRCS_UNSAFE_ByteOffset:
+                            case NI_SRCS_UNSAFE_IsAddressGreaterThan:
+                            case NI_SRCS_UNSAFE_IsAddressLessThan:
+                            case NI_SRCS_UNSAFE_IsNullRef:
+                            case NI_SRCS_UNSAFE_Subtract:
+                            case NI_SRCS_UNSAFE_SubtractByteOffset:
+                            {
+                                // These are effectively primitive binary operations so the
+                                // handling roughly mirrors the handling for CEE_ADD and
+                                // friends that exists elsewhere in this method
+
+                                if (!preciseScan)
```

Comment:
I assume this path is added to improve the inliner used in R2R? I don't think we care much about it honestly. And NativeAOT for perf critical path uses new one (--Ot). All other places with `!preciseScan` were added for backward compatibility

Reply:
Done for consistency with the other path. Figure it's better to match today and we can remove the handling altogether in a separate PR if it's no longer important/relevant.